### PR TITLE
Smbv2

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33284
fixes: https://github.com/demisto/etc/issues/26454
## Description
Added the SMBv2 integration to support the SMB3 protocol (the previous version supports SMB1 and SMB2).

## TDL
- [X] Find a solution with @Tomer-M to the connection restrictions. so we could run the TPB on our nightly builds Related: https://github.com/demisto/etc/issues/26454#issuecomment-703456278

## Screenshots
SMB_v2-test_Sun_Feb_07_2021 (1)

### CODE REVIEW

File: Packs/SMB/Integrations/SMB_v2/SMB_v2.yml [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    description: Removes a directory from the given path.
+  dockerimage: demisto/smbprotocol:1.0.0.16104
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+tests:
+- SMB_v2-Test
+fromversion: 5.5.0
```
Comment (yaakovi) [inline Packs/SMB/Integrations/SMB_v2/SMB_v2.yml:182]:
```suggestion
fromversion: 5.0.0
```

File: Packs/SMB/Integrations/SMB_v2/SMB_v2.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    hostname = args.get('hostname')
+    username = args.get('username')
+    password = args.get('password')
+    path = handle_path(args.get('path'))
+    path = os.path.join(hostname or client.hostname, path)
+
+    client.create_session(hostname, username, password)
+    entries = list(scandir(path))
+    files = [entry.name for entry in entries if entry.is_file()]
+    dirs = [entry.name for entry in entries if entry.is_dir()]
```
Comment (yaakovi) [inline Packs/SMB/Integrations/SMB_v2/SMB_v2.py:124]: switch to a single loop

File: Packs/SMB/TestPlaybooks/playbook-SMB_v2-test.yml [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+        "height": 2135,
+        "width": 492.5,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs: []
+outputs: []
+fromversion: 5.5.0
```
Comment (yaakovi) [inline Packs/SMB/TestPlaybooks/playbook-SMB_v2-test.yml:473]:
```suggestion
fromversion: 5.0.0
```

**Review by yaakovi — APPROVED** [review]
🆒

**Comment by kirbles19** [comment]
@Bargenish Done.

**Pull request merged**